pkg/client/v1/kscloudapi.go

```diff
@@ -310,15 +310,23 @@ func (api *KSCloudAPI) ListControls() ([]string, error) {
 }
 
 // SubmitReport uploads a posture report.
-func (api *KSCloudAPI) SubmitReport(report *PostureReport) error {
+func (api *KSCloudAPI) SubmitReport(report *PostureReport) (string, error) {
```

Comment:
This change is needed so we can use this function in Kubescape instead of this:
https://github.com/kubescape/kubescape/blob/master/core/pkg/resultshandling/reporter/v2/reporteventreceiver.go#L241C7-L241C7